### PR TITLE
Refixing the low tech shortsword.

### PR DESCRIPTION
**Pull request by bgaurons**

### CODE REVIEW

File: Library/Equipment/Low Tech.eqp [hunk 1]
```diff
@@ -12998,7 +12998,7 @@
 		<melee_weapon>
 			<damage>sw-1 cut</damage>
 			<strength>7</strength>
-			<usage>Swung</usage>
+			<usage>Swung - Shortsword</usage>
```
Comment (richardwilkes) [inline Library/Equipment/Low Tech.eqp]: Why a qualifier on the usage? It already shows the name of the weapon in the first column of the weapon table... and this isn't a shortsword, anyway, even if one of the skills that can be used for it is.

File: Library/Equipment/Low Tech.eqp [hunk 2]
```diff
@@ -13030,7 +13030,7 @@
 		<melee_weapon>
 			<damage>thr imp</damage>
 			<strength>7</strength>
-			<usage>Thrust</usage>
+			<usage>Thrust - Shortsword</usage>
```
Comment (richardwilkes) [inline Library/Equipment/Low Tech.eqp]: Same comment about the qualifier

Comment (richardwilkes) [inline Library/Equipment/Low Tech.eqp]: Maybe I just don't grasp what this is trying to do, as I see more below where you are treating the weapon differently based on knife/shortsword, which doesn't make any sense to me.

**Comment by angelforest** [comment]
Just for the reference, LT67 - Long Knife is under Knife skill, LT69 - Long Knife is under Shortsword skill. I don't like it either but I don't see other solution aside from making "Long Knife as Shortsword" as separate piece of equipment.

**Pull request merged**